### ISSUE TITLE
kuberlr 0.4.3 no longer supports global kubectl options

### ISSUE THREAD

**Issue by jandubois** [flavio/kuberlr#45]
This worked fine in 0.4.2 (`kubectl` is a symlink to `kuberlr-0.4.2`, `./kubectl` is a symlink to `kuberlr-0.4.3`):

```console
$ kubectl --context rancher-desktop get no
NAME                   STATUS   ROLES                  AGE   VERSION
lima-rancher-desktop   Ready    control-plane,master   24m   v1.27.5+k3s1

$ ./kuberlr version
kuberlr version: 0.4.3 (tagged as "v0.4.3") 2023-09-13T07:38:14Z go1.21.0

$ ./kubectl --context rancher-desktop get no
flag provided but not defined: -context
Usage of ./kubectl:
  -add_dir_header
    	If true, adds the file directory to the header
[...]
```

Same thing happens for other global options, e.g. `./kubectl -n kube-system get po` breaks the same way.

**Comment by flavio** [comment]
@jandubois I just pushed a fix for that on the master branch. Can you test the fix by building kuberlr from the `master` branch? I did a quick test and it seems to be working fine now on my machine

**Comment by flavio** [comment]
Once verified I'll be happy to publish 0.4.4

**Comment by jandubois** [comment]
Thanks, seems to be working fine now.

**Comment by flavio** [comment]
Closing, this is fixed with the 0.4.4 release: https://github.com/flavio/kuberlr/releases/tag/v0.4.4
